SharedPackages/BrowserServicesKit/Tests/PrivacyDashboardTests/PrivacyDashboardControllerTests.swift

```diff
@@ -289,7 +289,7 @@ class MockWebView: WKWebView {
             capturedJavaScriptString = javaScriptString
             expectation.fulfill()
         }
-        return try await super.evaluateJavaScript(javaScriptString)
+        return try await super.evaluateJavaScript(javaScriptString) as Void? as Any
```

Comment:
Need to come up with a better fix for this, the issue is that we declare this function ourselves and it causes ambiguity when compiling. This technically works for the tests but could bite people later.